storage/postgresql/query.go

```diff
@@ -136,32 +165,11 @@ func (b PostgresBackend) QueryEvents(ctx context.Context, filter *nostr.Filter)
 		params = append(params, filter.Limit)
 	}
 
-	query := b.DB.Rebind(`SELECT
+	query := sqlx.Rebind(sqlx.BindType("postgres"), `SELECT
```

Comment:
I changed this to use the exported `Rebind` function so we didn't have to take the `*sqlxDB` dependency in this function. Functionally this is the exact same underlying code, I verified that internally sqlx.DB.Rebind is just calling this exported function using the driver name.